Kernel/API/Syscall.h

```diff
@@ -51,7 +51,6 @@ enum class NeedsBigProcessLock {
     S(allocate_tls, NeedsBigProcessLock::Yes)              \
     S(anon_create, NeedsBigProcessLock::No)                \
     S(annotate_mapping, NeedsBigProcessLock::No)           \
-    S(beep, NeedsBigProcessLock::No)                       \
```

Comment:
Commit title nitpick: We aren't just removing the syscall, we are replacing it with a device.